src/Plugin/Block/AlertBannerBlock.php

```diff
@@ -65,6 +66,46 @@ public static function create(ContainerInterface $container, array $configuratio
     );
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return [
+      'include_types' => [],
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function blockForm($form, FormStateInterface $form_state) : array {
+    $form = parent::blockForm($form, $form_state);
+    $type_storage = $this->entityTypeManager->getStorage('localgov_alert_banner_type');
+    $config = $this->getConfiguration();
+    $config_options = [];
+    foreach ($type_storage->loadMultiple() as $id => $type) {
+      $config_options[$id] = $type->label();
+    }
+    $form['include_types'] = [
+      '#type' => 'checkboxes',
+      '#options' => $config_options,
+      '#title' => $this->t('Display types'),
+      '#description' => $this->t('If no types are selected all will be displayed.'),
+      '#default_value' => !empty($config['include_types']) ? $config['include_types'] : [],
+    ];
+    return $form;
+
```

Comment:
I thought coding standards check would pick this empty line up :)

Reply:
Added commit to remove, lets see what test says